src/NearOrg/FoundersPage.jsx

```diff
@@ -0,0 +1,277 @@
+  background-color: var(--background-color);
+  position: relative;
+  padding: 160px 24px;
+  overflow: hidden;
+
+  @media (max-width: 900px) {
+    padding: var(--section-gap) 24px;
+  }
+`;
+
+const Container = styled.div`
+  display: flex;
+  max-width: 1224px;
+  margin: 0 auto;
+  gap: ${(p) => p.gap ?? "var(--section-gap)"};
+  flex-direction: column;
+  align-items: ${(p) => (p.center ? "center" : undefined)};
+  justify-content: ${(p) => (p.center ? "center" : undefined)};
+  text-align: ${(p) => (p.center ? "center" : undefined)};
+`;
+
+const Pattern = styled.div`
+  width: 100%;
+  min-height: 540px;
+  display: flex;
+  align-items: center;
+  background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJYAAACWCAYAAAA8AXHiAAAACXBIWXMAABYlAAAWJQFJUiTwAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAGeSURBVHgB7doxTisxEAbgeY/mvQro6NiSDo6QkpJbcA2OwjWooKQMJ2DpKENJBV7FEYoBeQSIZr9PGk2cItWvsdfZnSBjKHVf6rnUbdD1N8g4K7VX6jhIEaycofaTIEWwcoam0yFYOYe179WiQ7Byhk8+8wnB6munlHNWgmD1tUGyFSYIVl8bJFcOCYLV106s/aBrJ2hNE+qo1GmpRanz2J5aB6X+x/oQv/l+FWz5E/O1iHU4pom0W/u0/uoZahnrgN2VGuv6Jpidl1+o2T5BznkrfKj9MdZT6l9836r+3k2pq1KXMVNz3gpbU7hOmj49AQ7x/lJ0WWsK5xhv2+AYkHQR29vbddDluqFvbNZPQZdg9S07az4gWH3tHZVgJQhW3xjb4XIZyo+Z3nffHN79CZ1gYuXc1b4KEytFsHLGptMhWDlj7Q9BimDlbJ4Ex4AftggHdwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIpXoUVLSWulnzoAAAAASUVORK5CYII=");
+  background-size: 75px 75px;
+  background-repeat: repeat;
+  background-position: center top;
+
+  @media (max-width: 900px) {
+    min-height: 390px;
+  }
+`;
+
+const PatternContent = styled.div`
+  padding: 1rem;
+  max-width: 496px;
+  margin: 0 auto;
+  background-color: var(--background-color);
+  display: flex;
+  align-items: center;
+  min-height: 260px;
+
+  @media (max-width: 900px) {
+    min-height: 0px;
+  }
+`;
+
+const IconCircle = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 64px;
+  height: 64px;
+  border-radius: 100%;
+  border: 1px solid var(--sand11);
+
+  i {
+    color: var(--sand11);
+    font-size: 32px;
+  }
+`;
+
+const Resource = ({ name, description, icon, url, target }) => (
+  <Widget
+    src="${REPL_ACCOUNT}/widget/NearOrg.Card"
+    key={name}
+    props={{
+      as: "a",
+      href: url,
+      target,
+      rel: "noopener noreferrer",
+      alt: name,
+      children: (
+        <>
+          <IconCircle>
+            <i className={`ph-duotone ${icon}`} />
+          </IconCircle>
+
+          <Flex direction="column" gap="16px">
+            <Text size="text-l" fontWeight="500">
+              {name}
+            </Text>
+            <Text size="text-s">{description}</Text>
+          </Flex>
+        </>
+      ),
+    }}
+  />
+);
+
+const selfServeResources = [
+  {
+    name: "Content Repository",
+    description:
+      "As a founder, I need to be able to access a repository housing templates, guides and resources essential for my journey.",
```

Comment:
hey @LCCCunningham  what text would you like us to use as a short description under each of these links?
<img width="341" alt="Screen Shot 2024-02-16 at 11 26 03 AM" src="https://github.com/near/near-discovery-components/assets/1915116/b925189a-6099-4bda-a339-6655f5b4eaf0">
 cc @shelegdmitriy

Reply:
@charleslavon I think you put the wrong screenshot here